<launch>
  <arg name="model_name" default="text-davinci-003"/>
  <arg name="generation_name" default="inspection"/>
  <arg name="wrong_raw_indices_string" default="1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30"/>

  <node name="behavior_tree_generation" pkg="behavior_tree_generation" type="behavior_tree_second_generation.py" output="screen">
    <param name="generation_name" value="$(arg generation_name)" />
    <param name="model_name" value="text-davinci-003" />
    <param name="wrong_raw_indices_string" value="$(arg wrong_raw_indices_string)"/>
  </node>

</launch>
